extensions/src/main/java/org/apache/falcon/extensions/store/ExtensionStore.java

```diff
@@ -250,6 +250,20 @@ public String getExtensionResource(final String resourcePath) throws StoreAccess
         return extesnionList;
     }
 
+    public String deleteExtensionMetadata(final String extensionName) throws StoreAccessException{
+        ExtensionType extensionType = AbstractExtension.isExtensionTrusted(extensionName) ? ExtensionType.TRUSTED
+                : ExtensionType.CUSTOM;
+        if (extensionType.toString().equalsIgnoreCase(ExtensionType.TRUSTED.toString())){
```

Comment:
You can directly do an enum comparison. No need to convert to string.

extensionType.equals(ExtensionType.TRUSTED)